### PR TITLE
middleware and i18n

### PR DESCRIPTION
**Pull request by BunnyPi04**
Check middle ware

### CODE REVIEW

**Comment by BunnyPi04** [comment]
ready

File: app/Http/Middleware/CheckAdmin.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if (Auth::check() && Auth::user()->level == config('const.roleAdmin'))
+        {
```
Comment (trinhductoan) [inline app/Http/Middleware/CheckAdmin.php]: conventions

File: app/Http/Middleware/CheckLogout.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if (Auth::guest())
+        {
```
Comment (trinhductoan) [inline app/Http/Middleware/CheckLogout.php]: convention

File: resources/views/backend/users/profile.blade.php [hunk 3]
```diff
@@ -73,8 +70,8 @@
                                 <div class="content" id="panel2-3">
                                     <h4 class="coral">{{ trans('messages.change-pass') }}</h4>
                                     <hr />
-                                    {{ Form::password('newpassword', array('id' => 'Password2', 'placeholder' => Lang::get('messages.new-pass'), 'required' => 'true')) }}
-                                    {{ Form::password('confirmpassword', array('id' => 'Password3', 'placeholder' => Lang::get('messages.confirm-pass'), 'required' => 'true')) }}
+                                    {{ Form::password('newpassword', array('id' => 'Password2', 'placeholder' => Lang::get('messages.new-pass'))) }}
```
Comment (trinhductoan) [inline resources/views/backend/users/profile.blade.php]: `Password2`

**Pull request merged**